### PR TITLE
Workshop update

### PR DESCRIPTION
**Pull request by jburel**
Update the entry for the workshop
cc @pwalczysko @will-moore @jrswedlow

### CODE REVIEW

File: events/omero-users-workshop-dundee-april-2019.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
      <hr class="whitespace">
         <div class="row">
             <div class="column">
-                <h2 class="event-day">Registration is now Open</h2>
+                <h2 class="event-day">Registration is now closed</h2>
                 <p class="event-day-description">To register your interest, click on the following link <a href="https://docs.google.com/forms/d/1Rup2R9DCR0OeBHd8PU3NVIb0BGX64OJ6aeu36v0CiPw" target="_blank">Register</a></p>
```
Comment (mtbc) [inline events/omero-users-workshop-dundee-april-2019.html]: Maybe no longer need link?

Comment (jburel) [reply events/omero-users-workshop-dundee-april-2019.html]: done

**Comment by pwalczysko** [comment]
I do not see the Oxford workshop in this list, although the other entries from this PR seem to have been built successfully. 
I mean

Screen Shot 2019-04-24 at 10 53 33

(Oxford missing) 

but

Screen Shot 2019-04-24 at 10 55 21
 (Oxford present as expected, shows that the PR has been included).

**Comment by jburel** [comment]
@pwalczysko done

**Comment by pwalczysko** [comment]
Works as expected. 

I just realized, we have the ELMI workshops still to do - but I guess this can be done in another PR.

File: index.html [hunk 2]
```diff
@@ -12,6 +12,8 @@ <h1 class="hero-main-header small-12">Next Workshop</h1>
                     </div>
                     <h2 class="hero-main-header small-12">Upcoming Workshops</h2>
                     <div align="center">
+                    <p align="left" class="small-10 small-offset-1 medium-6 medium-offset-3">05-06 June in Brno, Elmi meeting -- <a href="{{ site.baseurl }}/events/omero-users-workshop-brno-june-2019.html" style="color:white">See program</a></p>
```
Comment (pwalczysko) [inline index.html]: Elmi -> ELMI

see for example 

Screen Shot 2019-04-24 at 11 43 46

**Comment by pwalczysko** [comment]
Again, missing in Events list
Screen Shot 2019-04-24 at 11 46 49

**Comment by jburel** [comment]
Did you rebuild? 
refresh it should be in

**Comment by pwalczysko** [comment]
@jburel \yes, sorry, refresh was needed.

Also the ELMI capitalization is now built and reads well.

Ready to merge

**Pull request merged**